api/envoy/config/core/v3/health_check.proto

```diff
@@ -103,8 +111,16 @@ message HealthCheck {
     // [#not-implemented-hide:] HTTP specific payload.
     Payload send = 3;
 
-    // [#not-implemented-hide:] HTTP specific response.
-    Payload receive = 4;
+    // HTTP specific expected response.
```

Comment:
This is a repeated field, but IIUC the doc refers to it as a single entry.

Reply:
Changed the doc. Thanks!